.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Pconsume-incrementals
```

Comment:
Does it really make sense to enable this for all executions? 
I would expect if you are already changing the pom.xml it's not a problem to add this on your branch

Reply:
Why not?

I case you are fearing additional queries to the "incrementals" repository:
Having a look at the pom via `mvn help:effective-pom` shows the three configured repositories "repo.jenkins-ci.org", "incrementals", "central". Dependency resolution uses them in this order. Therefore the "incrementals" repository is only queried if the dependency is not found in "repo.jenkins-ci.org".

Reply:
I was more worried about accidentally releasing a version depending on incremental version.

I think it would be better to add `-Pconsume-incrementals` in the Jenkinsfile for PR builds instead, but this can be done as another improvement later.